Server.Core/Server.cs

```diff
@@ -60,7 +60,9 @@ public void HandleClients()
         private void RespondToClient()
         {
             byte[] clientMessage = Read();
-            Request request= new Request(clientMessage);
+            Console.WriteLine(Encoding.UTF8.GetString(clientMessage));
+            RequestBuilder requestbuilder = new RequestBuilder();
+            Request request = requestbuilder.BuildRequestFromWholeMessage(clientMessage);
```

Comment:
Not sure `BuildRequestFromWholeMessage` is descriptive enough. Is the "whole message" included in a string, a byte[], a stream, etc...?